package.json

```diff
@@ -43,7 +43,7 @@
     "codecov": "1.0.1",
     "commitizen": "2.9.5",
     "cross-env": "3.1.4",
-    "cz-lerna-changelog": "1.0.0",
+    "cz-lerna-changelog": "0.3.2",
```

Comment:
Haha yeah, sorry forgot to explain why. There is no version `1.0.0` the latest is `0.3.2`.